python/ray/serve/api.py

```diff
@@ -249,6 +257,10 @@ def create_backend(
                 update={"internal_metadata": metadata})
         else:
             raise TypeError("config must be a BackendConfig or a dictionary.")
+        if backend_config.user_config and not inspect.isclass(func_or_class):
```

Comment:
Sounds good, we do that check in `backend_worker` currently.  That way, it's checked upon calling `create_backend`, as well as upon calling `update_backend_config`. I will move the `inspect.isclass` check into `backend_worker` as well, since that should also happen for both `create_backend` and `update_backend_config`.